libnd4j/include/ops/declarable/generic/images/yuvToRgb.cpp

```diff
@@ -0,0 +1,56 @@
+/*******************************************************************************
+ * Copyright (c) 2019 Konduit K.K.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Apache License, Version 2.0 which is available at
+ * https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ ******************************************************************************/
+
+//
+// @author Oleh Semeniv (oleg.semeniv@gmail.com)
+// 
+
+#include <ops/declarable/headers/images.h>
+#include <ops/declarable/CustomOperations.h>  
+#include <helpers/ConstantTadHelper.h>
+#include <execution/Threads.h>
+
+namespace nd4j {
+namespace ops {
+
+CONFIGURABLE_OP_IMPL(yuv_to_rgb, 1, 1, false, 0, 0) {
```

Comment:
This operation can be used inplace as far as i understand. So, why "false" here?